### DISCUSSION TITLE
Customizing Presets - Remove mouse events

### DISCUSSION THREAD

**Discussion by Fedinjo** [matteobruni/tsparticles#4879] [Q&A] [answered]
Hi,

as the documentation illustrates, we are able to customize presets overriding the options, but what if the preset (for example the "firefly" preset) uses mouse events and we want to replace the mousehover event with a normal autoplay?
For example, on mobile the mousehover event is only triggered when scrolling, so it would be cool if you can make it autoplay.

**Comment by matteobruni** [comment, marked as answer]
Hi,

There's an old sample here: https://codepen.io/matteobruni/pen/ExPoybM

It simulates mouse events programmatically, or you can disable mouse events and use emitters for creating other effects. This sample contains many emitters samples: https://codepen.io/matteobruni/pen/wvKKXXa, or you can see them in the fireworks preset.
